contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -771,6 +783,15 @@ define_custom_variables(void)
 							   GUC_NOT_IN_SAMPLE | GUC_NO_RESET_ALL,
 							   NULL, NULL, NULL);
 
+	DefineCustomStringVariable("babelfishpg_tsql.psql_logical_babelfish_db_name",
```

Comment:
why are we using TSQL option to set option on PG endpoint? wouldn't a PG side guc will make more sense?

Reply:
This GUC will only be used by TSQL objects, hence it will be used only when TSQL extension is installed. If we include it at PG level then it will be there even if TSQL is not installed.